### ISSUE TITLE
[a11y] New should be at the bottom of any QCBX suggestions

### ISSUE THREAD

**Issue by grantfitzsimmons** [specify/specify7#1546]
![image](https://user-images.githubusercontent.com/37256050/168086512-73d74f45-7f59-40a9-b964-053ba0fb2b22.png)

It would be intuitive to have the 
![image](https://user-images.githubusercontent.com/37256050/168086629-5a5b284c-a0f8-4e14-b54a-927d7fbf9a11.png)
option at the bottom of the QCBX list at all times. Logically, this is where I would want to click add rather than to the right.
